src/workerd/api/data-url.c++

```diff
@@ -21,12 +22,16 @@ kj::Maybe<DataUrl> DataUrl::from(const jsg::Url& url) {
   // Per the fetch spec, it doesn't matter if the comma is within a quoted
   // string value in the MIME type... which is fun.
 
-  static const auto isAsciiWhitespace = [](auto c) {
-    return c == 0x09 /* tab */ ||
-          c == 0x0a /* lf  */ ||
-          c == 0x0c /* ff  */ ||
-          c == 0x0d /* cr  */ ||
-          c == 0x20 /* sp  */;
+  static constexpr std::array<uint8_t, 256> ascii_whitespace_table = []() consteval {
```

Comment:
Have you considered using kj::FixedArray here? Looks like at least some of its methods are constexpr and we generally try to lightly use the C++ STL. LGTM otherwise.

Reply:
Nice. I didn't know kj::FixedArray existed. Just updated now. I also updated the previous optimization to use fixed array as well. Appreciate the review @fhanau